chore: updated contributing guide (#377)
vueuse/vueuse: CONTRIBUTING.md

- Before you start working, it's better to open an issue to discuss first.
- The implementation should be placed under `packages/core` as a folder and exposing in `index.ts`

> - In the `core` package, try not to introduce 3rd party dependencies as this package is aimed to be as lightweight as possible.

Reviewer: ```suggestion
- In the `core` package, try not to introduce 3rd-party dependencies as this package is aimed to be as lightweight as possible.
```

Haha seems someday I was tried to add the dash but with my mind messed up 😂

> - If you'd like to introduce 3rd party dependencies, please contribute to add-ons.

Reviewer: ```suggestion
- If you'd like to introduce 3rd-party dependencies, please contribute to @vueuse/integrations or create a new add-on.
```

> - You can find the function template under `packages/core/_template/`, details explained in the [Function Foloder](#function-folder) section.

Reviewer: ```suggestion
- You can find the function template under `packages/core/_template/`, details explained in the [Function Folder](#function-folder) section.
```
